bigchaindb/tendermint/commands.py

```diff
@@ -34,9 +35,14 @@ def start():
     p_webapi.start()
 
     # start message
-    print(BANNER.format(bigchaindb.config['server']['bind']))
-
-    subprocess.Popen(['tendermint', 'node', '--consensus.create_empty_blocks=false'])
+    logger.info(BANNER.format(bigchaindb.config['server']['bind']))
+
+    if int(getenv('BIGCHAINDB_START_TENDERMINT', 1)):
```

Comment:
since there are no code bindings between tendermint and bdb servers I think It would be better if keep these decoupled.

Reply:
I agree, this is a temporary thing.

/cc @vrde